lib/Bio/EnsEMBL/DataCheck/Checks/CheckChar.pm

```diff
@@ -0,0 +1,85 @@
+=head1 LICENSE
+
+Copyright [2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CheckChar;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Test::DataCheck;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+    NAME        => 'CheckChar',
+    DESCRIPTION => 'Check that imported names/descriptions contains only supported characters',
+};
+
+sub tests {
+    my ($self) = @_;
+
+    # Test description length
+    my $sql_length = qq/
+        SELECT *
+        FROM phenotype
+        WHERE description IS NOT NULL
+        AND LENGTH(description) < 4
+    /;
+    my $desc_length = 'Phenotype description length';
+    my $diag_length = "Phenotype is suspiciously short";
```

Comment:
I think "Description is suspiciously short" would be a bit clearer.

Reply:
This was the message issued from original HC, just added length information.